network-api/networkapi/mozfest/blocks/carousel_text_block.py

```diff
@@ -0,0 +1,21 @@
+from wagtail import blocks
+
+from networkapi.wagtailpages.pagemodels import customblocks
+
+
+class CarouselTextBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(help_text="Heading for the block.", required=False)
+
+    text = blocks.RichTextBlock(features=["bold", "italic", "link"])
+    carousel_images = blocks.ListBlock(customblocks.ImageBlock(), max_num=4)
```

Comment:
**Nitpick** What do you think about grouping the definitions here similar to how they will show? In this case moving the `carousel_images` below all the other fields.

Reply:
:+1: good spot, thanks